tests/Fixtures/collection.json

```diff
@@ -59,7 +60,8 @@
                             "protocol": "http",
                             "host": "localhost",
                             "path": "api\/withResponseTag",
```

Comment:
It probably works the same, but I'm wondering why the `path` has its slash escaped (probably unnecessary, since it's a forward slash), while your raw URL doesn't. Any idea why?

Reply:
No, I'm not sure.

But I see that the last commit on path says "_Adjusting existing tests for new format and fixing encoding issue_". 

If there might be encoding-issues it may be best to encode my raw url as well?

Reply:
No worries. We'll leave it as is for now.